site/content/docs/11-deploying.md

```diff
@@ -55,10 +53,10 @@ Then you can set it using the environment variable, e.g.:
 SAPPER_TIMESTAMP=$(date +%s%3N) npm run build
 ```
 
-When deploying to [Now][], you can pass the environment variable into Now itself:
+When deploying to [Vercel], you can pass the environment variable into Now itself:
 
 ```bash
 now -e SAPPER_TIMESTAMP=$(date +%s%3N)
 ```
 
-[Now]: https://zeit.co/now
\ No newline at end of file
+[vercel]: https://vercel.com/home
```

Comment:
should this be `[Vercel]`? I don't know if it makes a difference. did you test rendering it?